lepton/image.go

```diff
@@ -77,11 +77,11 @@ func addHostName(m *fs.Manifest, c *types.Config) {
 	data := []byte("uniboot")
 	err := os.WriteFile(hostname, data, 0644)
 	if err != nil {
-		panic(err)
+		fmt.Printf("Failed save hostname tmp file, error: %s", err)
 	}
 	err = m.AddFile("/proc/sys/kernel/hostname", hostname)
 	if err != nil {
-		panic(err)
+		fmt.Printf("Failed add hostname, error: %s", err)
 	}
```

Comment:
since this was a panic need to exit here

Reply:
done